src/managers/GuildEmojiRoleManager.js

```diff
@@ -71,13 +75,17 @@ class GuildEmojiRoleManager {
   remove(roleOrRoles) {
     if (roleOrRoles instanceof Collection) return this.remove(roleOrRoles.keyArray());
     if (!Array.isArray(roleOrRoles)) return this.remove([roleOrRoles]);
-    roleOrRoles = roleOrRoles.map(r => this.guild.roles.resolveID(r));
 
-    if (roleOrRoles.includes(null)) {
-      return Promise.reject(new TypeError('INVALID_TYPE', 'roles', 'Array or Collection of Roles or Snowflakes', true));
+    const resolvedIDs = [];
```

Comment:
```suggestion
    const resolvedRoles = [];
```
This should be consistent with the naming in the `add` method